### PR TITLE
Wallet/Transaction List rows, crypto amount to show correct decimal places

### PR DESCRIPTION
**Pull request by amanalili**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android

### CODE REVIEW

File: src/util/utils.js [hunk 1]
```diff
@@ -320,7 +320,7 @@ export function precisionAdjust(params: PrecisionAdjustParams): number {
 
   if (bns.lt(precisionAdjust, '1')) {
     const fPrecisionAdject = parseFloat(precisionAdjust)
-    let order = 2 + Math.floor(Math.log(fPrecisionAdject) / Math.LN10 - 0.000000001) // because float math sucks like that
+    let order = 3 + Math.floor(Math.log(fPrecisionAdject) / Math.LN10 - 0.000000001) // because float math sucks like that
```
Comment (amanalili) [inline src/util/utils.js:323]: use this instead

  const exchangeRateString = bns.mul(exchangeRateOrderOfMagnitude.toString(), bns.mul(params.secondaryExchangeMultiplier, 10))

File: src/util/utils.js [hunk 2]
```diff
@@ -604,10 +604,31 @@ export function debounce(func: Function, wait: number, immediate: boolean): any
   }
 }
 
+export function getCustomTokenDenomination(currencyCode: string, settings: Object) {
```
Comment (amanalili) [inline src/util/utils.js:607]: create test

File: src/util/utils.js [hunk 3]
```diff
@@ -604,10 +604,31 @@ export function debounce(func: Function, wait: number, immediate: boolean): any
...
+  const customTokenCurrencyInfo = settings.customToken.find(token => token.currencyCode === currencyCode)
+  return customTokenCurrencyInfo ? customTokenCurrencyInfo.denominations[0] : emptyEdgeDenomination
+}
+
 export function getDenomination(currencyCode: string, settings: Object): EdgeDenomination {
-  const denominationMultiplier = settings[currencyCode].denomination
-  const denomination = settings[currencyCode].denominations.find(denomination => denomination.multiplier === denominationMultiplier)
-  return denomination || emptyEdgeDenomination
+  const currencyInfo = settings[currencyCode]
```
Comment (amanalili) [inline src/util/utils.js:613]: create test

**Comment by amanalili** [comment]
fixed comments

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**